lib/oli/delivery/sections.ex

```diff
@@ -2400,7 +2403,9 @@ defmodule Oli.Delivery.Sections do
     objectives_pages_map =
       DeliveryResolver.all_pages(section_slug)
```

Comment:
I think we want a specialized query that will pull all attached objectives, instead of fetching the revisions of all pages.  In a large course (with more than 500 or 600 pages) this approach may introduce problems.